client/src/livingdocs-component-app/app.html

```diff
@@ -56,23 +60,23 @@ <h4>${'livingdocsComponent.emptyList' & t}</h4>
           <div class="livingdocs-component-item-title">
             <h2 if.bind="title">${title}</h2>
           </div>
-          <preview-container ref="previewContainer" id="preview-container" width.bind="previewWidth" rendering-info.bind="renderingInfo"
-            target.bind="target">
+          <preview-container ref="previewContainer" id="preview-container" width.bind="previewWidth"
+            rendering-info.bind="renderingInfo" target.bind="target">
           </preview-container>
         </div>
-        <h2 class="livingdocs-component-item-preview__message" show.bind="selectedItemIndex === undefined">${'livingdocsComponent.selectGraphic' & t}</h2>
-        <h2 class="livingdocs-component-item-preview__message" show.bind="selectedItemIndex !== undefined && !selectedItems[selectedItemIndex].active">${'livingdocsComponent.inactiveGraphic' & t}</h2>
+        <h2 class="livingdocs-component-item-preview__message" show.bind="selectedItemIndex === undefined">${'livingdocsComponent.selectGraphic'
+          & t}</h2>
+        <h2 class="livingdocs-component-item-preview__message" show.bind="selectedItemIndex !== undefined && !selectedItems[selectedItemIndex].active">${'livingdocsComponent.inactiveGraphic'
+          & t}</h2>
       </div>
       <div class="livingdocs-component-item-controls-container">
         <div class="livingdocs-component-item-options">
-          <div if.bind="selectedItemIndex !== undefined && selectedItems[selectedItemIndex].active">
+          <div class="livingdocs-component-item-options-container" if.bind="selectedItemIndex !== undefined && selectedItems[selectedItemIndex].active">
             <form class="q-form">
-              <div class="q-form__checkbox" show.bind="displayOptionsSchema.properties" repeat.for="optionKey of displayOptionsSchema.properties | keys">
-                <label>
-                  <input type="checkbox" checked.bind="selectedItems[selectedItemIndex].toolRuntimeConfig.displayOptions[optionKey]" change.delegate="loadPreview()">
-                  <span class="q-text">${displayOptionsSchema.properties[optionKey].title}</span>
-                </label>
-              </div>
+              <schema-editor schema.bind="displayOptionsSchema" data.bind="selectedItems[selectedItemIndex].toolRuntimeConfig.displayOptions"
+                change.call="loadPreview()">
+                <h6>${optionsSchema.title || 'Anzeigeoptionen'}</h6>
```

Comment:
I would remove this. We haven't had this title before and the screen real estate is pretty packed already.
Do you think it is really necessary for the user to understand what these options are for?

Reply:
Removed in https://github.com/nzzdev/Q-editor/pull/166/commits/a0f019d1664c5a0f0ee567a483472cce45bc0be8